web/mass_class/views.py

```
@@ -0,0 +1,57 @@
+# web/mass_class/views.py
+from django.contrib.auth.decorators import login_required
+from django.http import JsonResponse
```

Ruff (0.8.2)

`django.http.JsonResponse` imported but unused

Remove unused import: `django.http.JsonResponse`

(F401)